### PR TITLE
UIIN-2775 Create new settings for classification type sorting

### PR DESCRIPTION
**Pull request by BogdanDenis**
## Description
Create new settings for classification type sorting

## Screenshots

https://github.com/folio-org/ui-inventory/assets/19309423/0007ca75-9774-4382-9315-3e81cad90607

## Issues
[UIIN-2775](https://folio-org.atlassian.net/browse/UIIN-2775)

## Related PRs
https://github.com/folio-org/stripes-connect/pull/229
https://github.com/folio-org/stripes-smart-components/pull/1463

### CODE REVIEW

File: src/hooks/useClassificationIdentifierTypes/useClassificationIdentifierTypes.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  useOkapiKy,
+} from '@folio/stripes/core';
+
+import {
+  CQL_FIND_ALL,
+  LIMIT_MAX,
+} from '../../constants';
+
+const useClassificationIdentifierTypes = () => {
+  const ky = useOkapiKy();
```
Comment (mariia-aloshyna) [inline src/hooks/useClassificationIdentifierTypes/useClassificationIdentifierTypes.js:14]: Is this the same for all tenants?

File: src/settings/ClassificationBrowseSettings/ClassificationBrowseSettings.test.js [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+  <MemoryRouter>
+    <ClassificationBrowseSettings
+      {...defaultProps}
+      {...props}
+    />
+  </MemoryRouter>,
+  translationsProperties
+);
+
+describe('ClassificationBrowseSettings', () => {
```
Comment (mariia-aloshyna) [inline src/settings/ClassificationBrowseSettings/ClassificationBrowseSettings.test.js:64]: Add check for accessibility, please

**Comment by mariia-aloshyna** [comment]
Fix eslint warnings, please

File: src/hooks/useClassificationIdentifierTypes/useClassificationIdentifierTypes.js (hunk 1 above)
Comment (BogdanDenis) [reply src/hooks/useClassificationIdentifierTypes/useClassificationIdentifierTypes.js]: Updated requests to use central tenant

**Review by mariia-aloshyna — APPROVED** [review]

**Pull request merged**